Add doc changes related to top down API development feature (#5265)
wso2/docs-apim: en/docs/integrate/develop/creating-artifacts/creating-an-api.md

                Select this option to import an existing REST API configuration (XML definition) that was created using WSO2 Integration Studio.
            </td>
        </tr>
        <tr>

>             <th>

Author: Ack. Will change in the next PR along with the other screenshots

            </tr>
        </table>

    -   If you selected **Import API from API Manager** in the previous step, enter the following information:

>         <img src="{{base_path}}/assets/img/integrate/create_artifacts/new_api/import-api-dialog.png" width="500">

Reviewer: Add the image width as a %

        <table>
            <tr>
                <th>
                    Parameter
                </th>
                <th>
                    Description
                </th>
            </tr>
            <tr>
                <td>
                    Username
                </td>
                <td>
                    Username of the API Manager user.
                </td>
            </tr>
            <tr>
                <td>
                    Password
                </td>
                <td>
                    Password of the API Manager user.
                </td>
            </tr>
            <tr>
                <td>
                    API Manager host
                </td>
                <td>

>                     Host Url of the API Manager.

Reviewer: ```suggestion
                    Host URL of the API Manager.
```

                </td>
            </tr>
        </table>

>         After entering above values in the Import API from API Manager option wizard, click <b>List APIs</b> button and it will list down the APIs which are in WSO2 API Manager. Thereafter, choose one of the APIs from the API list.

Reviewer: ```suggestion
        After entering the above values in the <b>Import API from API Manager</b> option wizard, click <b>List APIs</b>. The list of APIs that are in WSO2 API Manager appear. Thereafter, select one of the APIs from the API list.
```
